04.build-guide/08.coding-standards/docs.md

```diff
@@ -363,7 +363,27 @@ Include guards should be in the following format: hifi_$BASENAME_h.
 
 ```
 
-#### 2.2.2 Include statements should be sorted and grouped
+#### 2.2.2 Source files must include their associated header first
+
+The associated header is always included first to guarantee is it self-sufficient.
```

Comment:
it is

Reply:
Also we should still probably have them grouped by external library and our libraries

Reply:
> Also we should still probably have them grouped by external library and our libraries
What do you mean by that? I kept the groupings in my example.

Reply:
Right, but you changed the title of the section to be specific to what you wanted to add